src/loki/status_action.cc

```diff
@@ -0,0 +1,16 @@
+#include "loki/worker.h"
+
+namespace valhalla {
+namespace loki {
+void loki_worker_t::status(Api&) const {
```

Comment:
Does this not need to return something like `odin::status` ? I guess its not clear to me why `loki` & `thor` don't return (even a dummy) status

Reply:
This is just me adding the endpoint. As there is a lot of room for discussion as to what should be in the endpoint I decided to just leave it blank. I suspect that what we want to do is change the proto to have a spot for a status object, then each piece of the pipeline can add information about itself. Loki can mention something about its status (maybe some statistics about requests its handled?) and Thor and Odin can do the same. I just felt like that was such a big question mark as to what should actually be in the status that I just wanted to add the hook here and then maybe discuss further what should be in that status elsewhere and in a separate PR. In fact maybe I shouldnt mark that issue as fixed by this until we at least put some status-y info into the return.

Anyway the design, once we have something to fill out in the proto, will be that each of these will modify the `Api` object to add its status information and then at the very end Odin will serialize it to json. I hope this makes sense!